portfolio/src/main/webapp/index.html

```diff
@@ -40,33 +40,23 @@ <h1>Governess Simpson</h1>
         I am a rising junior at the University of Minnesota, majoring in Industrial and Systems Engineering with a minor in 
          Computer Science! This summer, I will be a Program Manager intern at Microsoft. I aspire to become a Product Manager
          (Technical or Associate).</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-
-      <div id="greeting-container"></div>
-
     </div>
 
-        <div id="content2">
+    <div id="content2">
       <p class="intro">
         I am a rising junior at the University of Minnesota, majoring in Industrial and Systems Engineering with a minor in 
          Computer Science! This summer, I will be a Program Manager intern at Microsoft. I aspire to become a Product Manager
          (Technical or Associate).</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-
-      <div id="greeting-container"></div>
-
     </div>
 
-    <div id="letsconnect">
-      <p class="intro">
-        Linkedin:
-        Github:
-        Resume:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-
-      <div id="greeting-container"></div>
-
-    </div>
+    <div id="linkedin">
+        <a href="https://www.linkedin.com/in/governess-simpson/">
+        <img border="0" alt="https://www.linkedin.com/"
+         src="/images/linkedin-icon.png" width="100" height="100"></a></div>
+    <div id='github'>
+        <a href="https://github.com/GovernessS">
+       <img border="0" alt="https://github.com/"
+        src="/images/github-icon.png" width="100" height="100"></a></div>
```

Comment:
These changes looks like it is the same as in the other PR. You can drip this PR.